common/src/test/kotlin/com.netguru.multiplatformstorage/MultiplatformStorageTest.kt

```diff
@@ -0,0 +1,232 @@
+//        //then
+//        assertEquals(BOOLEAN, data)
+//    }
+//
+//    @Test
+//    fun `should return default boolean value when storage doesn't contain key`() {
+//        //given
+//        val defaultValue = false
+//        storage.putBoolean(BOOLEAN_KEY, BOOLEAN)
+//        //when
+//        val data = storage.getBoolean("key", defaultValue)
+//        //then
+//        assertEquals(defaultValue, data)
+//    }
+//
+//    @Test
+//    fun `should contains call return true when storage contains key`() {
+//        //given
+//        storage.putInt(INT_KEY, INT)
+//        //when
+//        val value = storage.contains(INT_KEY)
+//        //then
+//        assertEquals(true, value)
+//    }
+//
+//    @Test
+//    fun `should contains call return false when storage doesn't contain key`() {
+//        //given
+//        storage.putInt(INT_KEY, INT)
+//        //when
+//        val value = storage.contains(STRING_KEY)
+//        //then
+//        assertEquals(false, value)
+//    }
+//
+//    @Test
+//    fun `should get all data from multiplatform storage`() {
+//        //given
+//        with(storage) {
+//            putString(STRING_KEY, STRING)
+//            putInt(INT_KEY, INT)
+//            putLong(LONG_KEY, LONG)
+//            putFloat(FLOAT_KEY, FLOAT)
+//            putDouble(DOUBLE_KEY, DOUBLE)
+//            putBoolean(BOOLEAN_KEY, BOOLEAN)
+//        }
+//        //when
+//        val dataMap = storage.getAll()
+//        //then
+//        assertTrue {
+//            dataMap.containsKey(STRING_KEY)
+//            dataMap.containsKey(INT_KEY)
+//            dataMap.containsKey(LONG_KEY)
+//            dataMap.containsKey(FLOAT_KEY)
+//            dataMap.containsKey(DOUBLE_KEY)
+//            dataMap.containsKey(BOOLEAN_KEY)
+//            dataMap.containsKey(BOOLEAN_KEY)
+//        }
+//    }
+//
+//    @Test
+//    fun `should remove key from storage when remove called`() {
+//        //given
+//        storage.putString(STRING_KEY, STRING)
+//        //when
+//        storage.remove(STRING_KEY)
+//        //then
+//        assertFalse(storage.contains(STRING_KEY))
+//    }
+//
+//    @Test
+//    fun `should remove all keys from storage when clear called`() {
+//        //given
+//        with(storage) {
+//            putString(STRING_KEY, STRING)
+//            putInt(INT_KEY, INT)
+//            putLong(LONG_KEY, LONG)
+//            putFloat(FLOAT_KEY, FLOAT)
+//            putDouble(DOUBLE_KEY, DOUBLE)
+//            putBoolean(BOOLEAN_KEY, BOOLEAN)
+//        }
+//        //when
+//        storage.clear()
+//        //then
+//        assertFalse {
+//            with(storage) {
+//                contains(STRING_KEY)
+//                contains(INT_KEY)
+//                contains(LONG_KEY)
+//                contains(FLOAT_KEY)
+//                contains(DOUBLE_KEY)
+//                contains(BOOLEAN_KEY)
+//            }
+//        }
+//    }
+
+    companion object {
```

Comment:
🤔 lots of not meaningful things = lots of questions I suppose 🤔

Reply:
as above. We will take care of that in separate task